src/logging.h

```diff
@@ -94,6 +97,13 @@ namespace BCLog {
         /** Log categories bitfield. */
         std::atomic<uint32_t> m_categories{0};
 
+        //! Category-specific log level. Overrides the global `m_log_level`.
+        std::unordered_map<LogFlags, Level> m_category_log_levels;
+
+        //! If there is no category-specific log level, all logs with a severity
+        //! level lower than `m_log_level` will be ignored.
+        Level m_log_level{Level::Info};
```

Comment:
Should `DEFAULT_LOG_LEVEL` be used here?

```suggestion
        Level m_log_level{DEFAULT_LOG_LEVEL};
```

Reply:
SGTM, done